### PR TITLE
Feature/contentful integration

### PR DESCRIPTION
**Pull request by PandaGO13**

### CODE REVIEW

File: apps/website/src/app/page/Home/index.tsx [hunk 1]
```diff
@@ -2,141 +2,97 @@ import GridWrapper from '@app/elements/organisms/GridWrapper';
...
-            <Icon name="linkedin"  type='outline' url="https://www.linkedin.com/company/dynamic-devs/" />
-            <Icon name="instagram" type='outline' url="https://www.instagram.com/dynamicdevs/" />
-            <Icon name="youtube"  type='outline' url="https://www.youtube.com/channel/UCD2cpIbELBfK_-9p8PoCGWg" />
-            <Icon name="spotify" type='outline' url="https://open.spotify.com/show/4fvKHii2mWHkX8mz28klz8?si=8d16ad1538da4899" />
-            <Icon name="twitch" type='outline' url="https://www.twitch.tv/dynamicdevs" />
+            {
+              data.socialMedia?.map((socialMediaItem) => 
+                <Icon 
+                  key={socialMediaItem.id}  
+                  name={socialMediaItem.icon.name}
```
Comment (c1sar) [inline apps/website/src/app/page/Home/index.tsx:34]: Validar si llega el icon  ```socialMediaItem.icon.name```

File: apps/website/src/app/page/Home/index.tsx [hunk 2]
```diff
@@ -2,141 +2,97 @@ import GridWrapper from '@app/elements/organisms/GridWrapper';
...
+                {section.title}
+              </h2>
+            </div>
+            <div className='col-span-4 md:col-span-10 md:col-start-2 lg:col-span-6 lg:col-start-4'>
+              <div className='roots-gap grid-cols-4 md:grid-cols-10 lg:grid-cols-6 auto-rows-fr'>
+                {
+                  section.cards.map((card) => (
+                    <Card
+                      key={card.id}
+                      image={{ url: card.image.url, alt: card.image.alt }}
```
Comment (c1sar) [inline apps/website/src/app/page/Home/index.tsx:82]: image={{ url: card.image?.url, alt: card.image?.alt }}

**Review by c1sar — CHANGES_REQUESTED** [review]

File: apps/website/src/app/page/Home/index.tsx (hunk 1 above)
Comment (PandaGO13) [reply apps/website/src/app/page/Home/index.tsx]: Done

File: apps/website/src/app/page/Home/index.tsx (hunk 2 above)
Comment (PandaGO13) [reply apps/website/src/app/page/Home/index.tsx]: Done

**Review by c1sar — APPROVED** [review]

**Pull request merged**